# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/tests/shaders/struct.frag
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/tests/shaders/struct.frag b/third_party/rust/spirv-cross-internal/tests/shaders/struct.frag
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/tests/shaders/struct.frag
+++ /dev/null
@@ -1,18 +0,0 @@
-#version 310 es
-precision mediump float;
-
-struct W
-{
-    vec4 e;
-    vec4 f;
-    vec4 g;
-    vec4 h;
-};
-
-layout(location = 0) in W w;
-
-layout(location = 0) out vec4 color;
-
-void main() {
-    color = vec4(w.e.x, w.f.y, w.g.z, w.h.w);
-}